### PR TITLE
Rewrite the Russian text to feel more natural

### PR DESCRIPTION
**Pull request by andrewshadura**
Hi,

I’ve rewritten some of the passages in the Russian text to feel more natural while staying close to the original.

### CODE REVIEW

**Comment by andrewshadura** [comment]
I must admit the appendix still needs more work. In particular, I haven’t found an appropriate way to say *[present themselves] entirely willing*, since *полностью добровольно готовы к интимным отношениям* is not exactly the same and literally means *entirely willingly ready to intimacy* whereas in my view *entirely willing* doesn’t say anything about readiness.

File: index.ru.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ lang: ru
...
+Мы, нижеподписавшиеся, верим в необходимость цифровой автономии и важную роль, которую свобода пользователей играет в защите наших основных человеческих прав. Чтобы реализовать все преимущества, которые даёт свобода программного обеспечения, в сообществе должны произойти радикальные изменения. Мы верим в настоящее и будущее, где технологии расширяют возможности, а не угнетают людей. Мы знаем, что это возможно только в мире, где технологии построены так, чтобы уважать наши права на самых основных уровнях. Хотя эти идеи в той или иной форме были популяризированы Ричардом М. Столлманом, наши взгляды на многие вопросы разнятся. Мы не согласны его поведением и мнениями. Мы не признаем его как лидера вообще и лидера Фонда свободного ПО на сегодняшний день.
```
Comment (kinmoza) [inline index.ru.md:10]: Оригинал: "realize the **promise** of everything", каким тут боком появились "преимущества", я так и не понимаю. Фраза "We do not condone" означает "Мы не оправдываем", но никак не "согласны". "как лидера вообще и лидера Фонда свободного ПО на сегодняшний день." - стилистические нарушение.

File: index.ru.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ lang: ru
...
 
-Наша терпимость к отвратительным идеям и поведению RMS иссякла. Мы не можем и дальше позволять одному человеку разрушать смысл нашей работы. В наших общинах нет места для таких людей, как Ричард М. Столлман и мы не желаем продолжать страдать от его отвратительного поведения, давая ему руководящую роль или иным образом считать RMS и его вредную, опасную идеологию приемлемой.
+Наша терпимость к отвратительным идеям и поведению RMS иссякла. Мы не можем и дальше позволять одному человеку разрушать смысл нашей работы. Таким людям, как Ричард М. Столлман, не место в наших сообществах, и мы не желаем продолжать расхлёбывать последствия его отвратительного поведения, давать ему руководящую роль или иным образом делать вид, что RMS и его вредная, опасная идеология является приемлемой.
```
Comment (kinmoza) [inline index.ru.md:12]: "продолжать **расхлёбывать** последствия" - нарушение стилистики официального письма. Перефразирование моих слов, что приводит к искажению смысла и уничтожению выразительности перевода.

File: index.ru.md [hunk 3]
```diff
@@ -5,15 +5,15 @@ lang: ru
...
 
-Мы призываем к отстранению всех членов совета правления FSF. Это люди, которые поддерживали полномочия Ричарда М. Столлмана в течение многих лет. Они еще раз демонстрируют это, разрешая ему вернуться в Правление FSF. Пришло время RMS отойти от свободного программного обеспечения, технической этики, цифровых прав и сообществ, поскольку он не может обеспечить лидерство, в котором мы нуждаемся. Мы также призываем к отстранению Ричарда М. Столлмана от всех других руководящих должностей, включая проект GNU.
+Мы призываем к отстранению всех членов совета директоров FSF. Это люди, которые поддерживали Ричарда М. Столлмана в течение многих лет. Они еще раз демонстрируют это, разрешая ему вернуться в совет директоров FSF. Пришло время RMS отойти от свободного программного обеспечения, технической этики, цифровых прав и сообществ, поскольку он не может быть лидером, в котором мы нуждаемся. Мы также призываем к отстранению Ричарда М. Столлмана от всех других руководящих должностей, включая проект GNU.
```
Comment (kinmoza) [inline index.ru.md:14]: "может быть лидером" =/= "cannot provide the leadership", к чему тут изменения - я не пойму, очередное перефразирование слов и искажение перевода.

File: index.ru.md [hunk 4]
```diff
@@ -5,15 +5,15 @@ lang: ru
...
 
-Мы призываем тех, кто в состоянии сделать это, прекратить поддерживать Free Software Foundation. Отказывайтесь участвовать в проектах связанных с FSF и RMS. Не выступайте и не посещайте мероприятия FSF или мероприятия, которые приветствуют RMS и его бренд нетерпимости. Мы просим участников проектов FSF выступить против фанатизма и ненависти в своих же проектах. Делая это, расскажите сообществам и FSF, почему вы это делаете.
+Мы призываем тех, кто в состоянии сделать это, прекратить поддерживать Фонд свободного ПО. Отказывайтесь участвовать в проектах связанных с FSF и RMS. Не выступайте и не посещайте мероприятия FSF или мероприятия, которые поощряют RMS и его бренд нетерпимости. Мы просим участников проектов FSF выступить против фанатизма и ненависти в своих же проектах. Делая это, расскажите сообществам и FSF, почему вы это делаете.
```
Comment (kinmoza) [inline index.ru.md:16]: Опять же, искажение перевода и перефразирование. "cannot provide the leadership" =/= "которые поощряют RMS".

**Review by kinmoza — COMMENTED** [review]
Как переводчику я вам скажу, у вас это очень плохо получается. Да, вы нашли пару ошибок в пунктуации, но ваши "переделки" - это лишнее. Personally, I am against your edits and additions that distort the text and add errors in the official writing style.

**Comment by kinmoza** [comment]
@isagalaev, what you think about it?

**Comment by andrewshadura** [comment]
> Personally, I am against your edits and additions that distort the text and add errors in the official writing style

Well, I'm sorry if this is going to offend you, but the original text was hardly in Russian. It was English written in Russian words, especially so in the appendix.

I might have made a mistake or two but I think my edits were overall close to the English text in essence and much closer in spirit while in some cases using different phrasing.

**Comment by kinmoza** [comment]
Уважаемый, ваш перевод ни разу не близок к оригиналу, перевод фраз абсолютно не точен и искажен. Я лично против ваших правок в перевод, спасибо, что указали пунктуационные ошибки, но в целом - перевод вы сильно исказили.

**Comment by andrewshadura** [comment]
Thank you for your thoughtful reply addressing all of the points I made ❤️

**Comment by isagalaev** [comment]
> @isagalaev, what you think about it?

First, let us all come down :-)

On the subject, being a stylistic argument, it won't ever be resolved to the satisfaction of either side (as you guys know very well, I'm sure). I myself would certainly write yet another, vastly different variant :-) So I can only share my approach to reviews. I try to let the author's text stand as much as possible and only point out objective mistakes: typos or factually incorrect phrases (like the missing word for "the Board" in the main letter which made it sound like it called *all* of the members to resign). Everything else, as long as it sounds "Russian enough", I'm shutting up.

**Comment by ehashman** [comment]
@isagalaev @andrewshadura @kinmoza do we have consensus here?

**Comment by andrewshadura** [comment]
@ehashman, @kinmoza’s response was not very encouraging, I guess I’ll leave it up to him and @isagalaev.

I think the current translation needs some serious proofreading and stylistic editing by someone other than the person who wrote it, and this someone needs to use Russian on an everyday basis. Sadly, none of us three live in a country where Russian is spoken, so we’re all quite limited in this regard.

**Comment by isagalaev** [comment]
> Sadly, none of us three live in a country where Russian is spoken, so we’re all quite limited in this regard.

That's an incorrect conjecture. Despite living in U.S. I do speak Russian daily, and still have quite a few connections with Russians living in Russia.

To reiterate, I do agree the text could be written in an easier style. But a flame war in the comments is not going to do that, which is why I suggest leaving things as is. Yes, most Russians upon reading the text would conclude it was translated from a foreign language. That's hardly a secret anyway :-)

**Comment by andrewshadura** [comment]
> Despite living in U.S. I do speak Russian daily, and still have quite a few connections with Russians living in Russia.

I also do use Russian daily (although I don’t speak it every day), but I am certain I’m not as fluent as I was in it before I moved abroad, and I know I try to use English and Slovak constructs in Russian when I speak and write.

I wasn’t going on a flame war, I proposed a contribution to seriously improve the quality of the text, which was met with unwillingness to accept even the smallest bits of it. I’m sorry I have to reiterate what I said above, but I think it’d be better not to have any Russian text than what we have currently, as it’s barely readable and feels like it’s taken straight from Google Translate.

**Comment by ehashman** [comment]
@isagalaev are the proposed changes an improvement?

**Comment by isagalaev** [comment]
@ehashman from my point of view, a mixed bug. There's quite a lot of fixed grammar, but there are also gratuitous changes that don't make any difference, and also those that make it less correct.

**Comment by kinmoza** [comment]
> @andrewshadura can we limit the changes to fixed grammar?
@ehashman. I only agree with punctuation errors, otherwise these are meaningless edits with violations of the style and form of the letter.

**Comment by ehashman** [comment]
Glad we're in agreement then. @andrewshadura waiting on you :)

**Comment by kinmoza** [comment]
@ehashman, ok, check this out. #2549

**Comment by ehashman** [comment]
Superseded by #2549.

**Comment by andrewshadura** [comment]
I’d not close this issue just yet, I’d like to remove some controversial stylistic edits and rework the remaining ones.

**Comment by ehashman** [comment]
Feel free to reopen once you've rebased.

**Comment by andrewshadura** [comment]
It seems I cannot reopen it myself…

**Pull request closed without merging**